docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       db:
         condition: service_healthy
   db:
-    image: neo4j:3.4-enterprise
+    image: neo4j:3.5.26
```

Comment:
Is there a tag that would allow us to use any patch version of 3.5?

Reply:
There is. Why do you think a floating tag is better than a pinned version?

Reply:
This would allow us to not have to update the tag for every patch version release of Neo4j and patch versions should always be safe to upgrade to, especially in a dev environment.

Reply:
The neo4j release should be the same in all environments, unless we are considering upgrading prod environment. That is the reason why I want to take a look at the neo4j deployment code.
With that, there is no reason to update the tag if we do not plan to upgrade the prod.
On the other hand, a pinned version helps spin a reproducible environment for troubleshooting. This is similar to how we pin PyPi packages in requirements.txt files.

Reply:
Okay, no problem. We can keep it pinned.